server/pulp/server/db/connection.py

```diff
@@ -12,9 +12,10 @@
 from pymongo.errors import AutoReconnect, OperationFailure
 from pymongo.son_manipulator import NamespaceInjector
 
+from pulp.common import error_codes
```

Comment:
I think this gets grouped with the imports above since this module is not inside the same package (`pulp.server`) like the ones in this group.